### PR TITLE
updating spec to include rocmcc instead of clang

### PR DESCRIPTION
**Pull request by kab163**

### CODE REVIEW

**Comment by kab163** [comment]
Now my new CI jobs on corona and tioga work, but these default ones don't because they are still trying to reference clang@13 - https://lc.llnl.gov/gitlab/radiuss/Umpire/-/jobs/960940

**Comment by adrienbernede** [comment]
Exactly, and that’s why, from a project point of view, radiuss-spack-configs and radiuss-shared-ci updated are often tied together.

**Comment by adrienbernede** [comment]
We have some branches in codes pointing directly at `radiuss-shared-ci@main`. That’s my fault, merging into `main` is therefore not safe. I’m opening a release candidate for March 2023 and pointing this branch at it.

File: corona-build-and-test.yml [hunk 1]
```diff
@@ -112,9 +112,9 @@ status_pending:
     - export pipeline_status="pending"
     - *status_report
 
-rocm_5_1_1_clang_13_0_0:
+rocm_5_1_1_rocmcc_5_1_1:
```
Comment (adrienbernede) [inline corona-build-and-test.yml:115]:
```suggestion
rocmcc_5_1_1_hip:
```

File: tioga-build-and-test.yml [hunk 2]
```diff
@@ -112,9 +112,9 @@ status_pending:
     - export pipeline_status="pending"
     - *status_report
 
-rocm_5_2_3_clang_13_0_0:
+rocm_5_2_3_rocmcc_5_2_3:
```
Comment (adrienbernede) [inline tioga-build-and-test.yml:115]:
```suggestion
rocmcc_5_2_3_hip:
```

**Review by adrienbernede — COMMENTED** [review]
Let’s simplify the naming

**Comment by kab163** [comment]
Wow! This was great to wake up to haha thanks for getting this sorted out and merging! :)

**Pull request merged**